bmi-calculator.js

```diff
@@ -0,0 +1,14 @@
+function calculateBmi (weight, height) {
+    let bmi = weight / (height * height);
+    if (bmi <= 18.5) {
+        return "Underweight"
+    }
+    if (bmi <= 25.0) {
+        return "Normal"
+    }
+    if (bmi <= 30.0) {
+        return "Overweight"
+    }
+    else return "Obese"
```

Comment:
Always use curly bracets with if else statement
```suggestion
    else {
      return "Obese"
    }
```

Reply:
apart from this, if you have the if else with return, then there is no reason to do the else at the end, just leave plain return after if else statement.